main.py

```diff
@@ -711,15 +704,37 @@ def handle_link_up(self, interface):
             now = time.time()
             if link.is_active() and \
                     now - last_status_change >= self.link_up_timer:
-                self.update_instance_metadata(link)
+                link.update_metadata('last_status_is_active', True)
+                self.topo_controller._update_link(
+                    link.id,
+                    {
+                        "$set": {
+                            "metadata.last_status_change": last_status_change,
+                            "metadata.last_status_is_active": True,
+                            "active": True,
+                        }
+                    },
+                )
```

Comment:
I think we shouldn't have to use mongo syntax in the main, it should be encapsulated in the controller/model.

Reply:
Indeed @ajoaoff good point, it's worth further encapsulating this part with a higher-level method instead instead of calling this more low level-ish `_update_link` that leak DB syntax outside of the controller, I forgot to further abstract this part, thanks for catching this,  done in these commits: [c846711](https://github.com/kytos-ng/topology/pull/84/commits/c846711aa16d6b0e344893dc379b91a84842056d) https://github.com/kytos-ng/topology/pull/84/commits/96afdfbc1f60361998f463007356467e78fd3a94 https://github.com/kytos-ng/topology/pull/84/commits/90f6d1707dee74232345995e764a9974b10a3b0e